db/elasticsearch/elasticsearch.yml

```diff
@@ -55,7 +55,7 @@ bootstrap:
 http:
   port: 9200
   cors:
-    allow-origin: http://localhost:50009,http://127.0.0.1:50009,https://opensource.appbase.io
+    allow-origin: http://localhost:40005,http://127.0.0.1:40005,https://opensource.appbase.io
```

Comment:
This is still hardcoded, but should be switched with a variable at some point to enable future changes to ports